Automattic/newspack-migration-ui
# feat(admin_menu): add admin menu

### All Submissions:






### Changes proposed in this Pull Request:



Adds an admin menu for migration handyness for posts

### How to test the changes in this Pull Request:

1. Read the docs/admin-menu.md file to see how to set the constants needed
2. Visit any post and click the menu with the shark.
3. Check that copy post id works
4. Check that you are taken to the correct post on live and on staging when clicking those menu items.

### Other information:
